qiskit/visualization/circuit_visualization.py

```diff
@@ -332,6 +332,9 @@ def _text_circuit_drawer(
     text_drawing.vertical_compression = vertical_compression
 
     if filename:
+        if not filename.endswith(".txt"):
+            logger.warning("WARNING Filename extension is not .txt. Please use a .txt file extension to save in text format.")
```

Comment:
Should we call this 'WARNING' or 'ERROR' in the message? I am a bit inclined towards 'error' since we are not saving any file but no strong opinions there. Same in line 458.

Reply:
Even I think we should call this an ERROR instead of a warning. I'll change that.